stepper.md

```diff
@@ -1,64 +1,214 @@
+3  device number                           (0-9)
+4  num target byte1 LSB
+5  num target byte2 MSB
+6  speed LSB                               (steps per second)
+7  speed MSB
+8  [optional] accel LSB                    (acceleration in steps/sec^2)
+9  [optional] accel MSB
+10 [optional] decel LSB                    (deceleration in steps/sec^2)
+11 [optional] decel MSB
+12 END_SYSEX                               (0xF7)
+```
+
+**Stepper enable**
+```
+0  START_SYSEX                             (0xF0)
+1  Stepper Command                         (0x62)
+2  enable command                          (0x04)
+3  device number                           (0-9)
+4  END_SYSEX                               (0xF7)
+```
+
+**Stepper disable**
+```
+0  START_SYSEX                             (0xF0)
+1  Stepper Command                         (0x62)
+2  disable command                         (0x05)
+3  device number                           (0-9)
+4  END_SYSEX                               (0xF7)
+```
+
+**Stepper stop**
+```
+0  START_SYSEX                             (0xF0)
+1  Stepper Command                         (0x62)
+2  stop command                            (0x06)
+3  device number                           (0-9)
+14 END_SYSEX                               (0xF7)
+```
+
+**Stepper reply (sent when a move completes or stop is called)**
+```
+0  START_SYSEX                             (0xF0)
+1  Stepper Command                         (0x62)
+2  stop reply command                      (0x07)
+3  device number                           (0-9)
+4  position byte 1 LSB                            
+5  position byte 2 MSB                            
+6  END_SYSEX                               (0xF7)
+```
+
+**Stepper limit**
+```
+0  START_SYSEX                             (0xF0)
+1  Stepper Command                         (0x62)
+2  stop limit command                      (0x08)
+3  device number                           (0-9)
+4  lower limit pin number                  (0-127)
+5  lower limit pin number                  (0-127)
+6  END_SYSEX                               (0xF7)
+```
+
+**MultiStepper configuration**
+```
+0  START_SYSEX                              (0xF0)
+1  Stepper Command                          (0x62)
+2  config command                           (0x20)
+3  group number                             (0-4) (supports up to 5 groups)
+4  member 0x00 device number                (0-9)
+5  member 0x01 device number                (0-9)
+6  [optional] member 0x02 device number     (0-9)
+7  [optional] member 0x03 device number     (0-9)
+8  [optional] member 0x04 device number     (0-9)
+9  [optional] member 0x05 device number     (0-9)
+10 [optional] member 0x06 device number     (0-9)
+11 [optional] member 0x07 device number     (0-9)
+12 [optional] member 0x08 device number     (0-9)
+13 [optional] member 0x09 device number     (0-9)
+14 END_SYSEX                                (0xF7)
+```
+
+**MultiStepper to**
```

Comment:
Short answer: It's not supported by multiStepper. 

Slightly longer answer: I think the primary arrangement for multiStepper is for controlling an end effector that operates on a 2D plane or in a 3D space. In these systems users need to tween between two points in a linear fashion. This would always be done in a to(X, Y) or to(X, Y, Z) fashion.